option optimize_for = LITE_RUNTIME;

message CommonRankRequest
{
 required uint32 type = 2;
}
